### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by habasm**
### This PR is by @habasm of the project Leader board: setup project

In this activity we set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. We develop a first working version of the app following a wireframe, but without styling - just focus on functionality. In this activities, We consume the Leaderboard API using JavaScript async and await and add some styling.

## [live] here https://habasm.github.io/leadership-boared-habasm/dist/ 

image

### CODE REVIEW

**Review by Favourezeugwa — COMMENTED** [review]
Hi @habasm , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌❌❌

You have submitted a project, that doesn't ;

1. use Gitflow. You should create a development branch, make it default, then compare a new feature branch with development. In your case, you care comparing development with main, which is wrong. Kindly use this link 👉🏽  [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)

2. You are currently using the old microverse readme template. Kindly change it to the newly approved template.  Here is a link to the [new Readme template](https://github.com/microverseinc/readme-template) 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by habasm** [comment]
@Favourezeugwa thanks for the comments and I am able to adjust it.
# dev branch added and made it also default
# Professional readme file included

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import './style.css';
+import AddScores from './views/AddScore.js';
+import Footer from './views/Footer.js';
+import Modal from './views/Modal.js';
+import RecentScores from './views/RecentScores.js';
+
+const leaderContent = document.querySelector('.leaderboard-content');
+
+// update leaderboard content
+leaderContent.innerHTML = _ + RecentScores() + AddScores();
```
Comment (TedLivist) [inline src/index.js:11]: - [ ] See the error I get when I run your app.

  image

  You can fix this by importing `recentScores` correctly and using it as intended.

  ```
  import { recentScores } from './views/RecentScores.js';
  
  ... 
  
  // update leaderboard content
  leaderContent.innerHTML = _ + recentScores() + AddScores();
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @habasm,

Good job so far!
✔️ App is properly set up with webpack
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (habasm) [reply src/index.js:11]: ## Thanks @TedLivist for your comment, now it's fixed based on the recommendation. 😃

Comment (mckent05) [reply src/index.js:11]: - Hello @habasm, I am still experiencing the same issues experienced by the previous reviewer, please follow the advice of the previous reviewer 🙏 
ythrth

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @habasm,
 
# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there


 ## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by habasm** [comment]
# fixed
 Actually I am forgot to submit/pull the changed once.

**Comment by habasm** [comment]
@TedLivist @mckent05 @Favourezeugwa  Thanks for comment but consumes my submission number via simple fixation it can be fixed easily by yours no matter if you correct it and make it as optional. 😢 😞

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @habasm, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**